pyHTTPInterface/Exceptions.py,sha256=nGmsNMXss9QaBHDEoevqqaaogn3uFHpnxkdWkZWQu64,1935
pyHTTPInterface/Request.py,sha256=vA9Qss7YZcV28F2O55VLKUtVb99feqTNRRW0w4JaG0c,3127
pyHTTPInterface/Response.py,sha256=2XgAOnWTgDK_agpXm46PL8ZvxFXJSLW7hKnd6rCyN1Q,1957
pyHTTPInterface/__init__.py,sha256=NicwNi8uMFWr8jJ9mfZn4c161bQ4hRiePIhAccSBGfk,2353
pyHTTPInterface-0.1.0.dist-info/LICENSE.md,sha256=DF4r-Jbsz_ZASY4VjV0ZZVaaHYkH9a20BkVExpq8doE,10571
pyHTTPInterface-0.1.0.dist-info/METADATA,sha256=Qe4LJ8Ktmz7nQbvu_RMu5mULCmBF9kCb2WAlVIaGbxI,1544
pyHTTPInterface-0.1.0.dist-info/WHEEL,sha256=p46_5Uhzqz6AzeSosiOnxK-zmFja1i22CrQCjmYe8ec,92
pyHTTPInterface-0.1.0.dist-info/top_level.txt,sha256=Cz2Ckg47BL44l5mWVBORpyPDWmn_O7LCfuzzOR4QtDc,16
pyHTTPInterface-0.1.0.dist-info/RECORD,,
